allensdk/brain_observatory/behavior/data_files/eye_tracking_file.py

```diff
@@ -0,0 +1,64 @@
+import json
+from typing import Dict, Union
+from pathlib import Path
+
+from cachetools import cached, LRUCache
+from cachetools.keys import hashkey
+
+import pandas as pd
+
+from allensdk.brain_observatory.behavior.eye_tracking_processing import \
+    load_eye_tracking_hdf
+from allensdk.internal.api import PostgresQueryMixin
+from allensdk.internal.core.lims_utilities import safe_system_path
+from allensdk.brain_observatory.behavior.data_files import DataFile
+
+
+def from_json_cache_key(cls, dict_repr: dict):
+    return hashkey(json.dumps(dict_repr))
+
+
+def from_lims_cache_key(cls, db, ophys_experiment_id: int):
+    return hashkey(ophys_experiment_id)
+
+
+class EyeTrackingFile(DataFile):
+    """A DataFile which contains methods for accessing and loading
+    eye tracking data.
+    """
+
+    def __init__(self, filepath: Union[str, Path]):
+        super().__init__(filepath=filepath)
```

Comment:
I'm forgetting, but just to double check, the `DataFile` `__init__()` method calls the `load_data` staticmethod right?

Reply:
yes